src/mongoc/mongoc-uri.c

```diff
@@ -990,6 +1046,15 @@ mongoc_uri_unescape (const char *escaped_string)
 }
 
 
+const mongoc_read_prefs_t *
+mongoc_uri_get_read_prefs_t (const mongoc_uri_t *uri) /* IN */
```

Comment:
Since `mongoc_uri_get_read_prefs()` was unfortunately taken by the getter function for the tags `bson_t`, I added `_t` here on @hanumantmk's advice. Since this adds a public API function, should I include new documentation in this PR?

My second question: how should we go about deprecating (at least in documentation) the `mongoc_uri_get_read_prefs()` function? A cursory grep for "deprecated" in the project didn't turn up much.